test/cli.bats

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env bats
+
+load vars
+
+@test "cli: show info" {
+  run ./docker-machine_$PLATFORM-$ARCH
+  [ "$status" -eq 0  ]
+  [[ ${lines[0]} =~ "NAME:"  ]]
+  [[ ${lines[1]} =~ "Create and manage machines running Docker"  ]]
+}
+
+@test "cli: show active help" {
+  run ./docker-machine_$PLATFORM-$ARCH active -h
+  [ "$status" -eq 0  ]
+  [[ ${lines[3]} =~ "command active"  ]]
+}
+
+@test "cli: show config help" {
+  run ./docker-machine_$PLATFORM-$ARCH config -h
```

Comment:
Minor nit:

I think you could define a function somewhere, say `machine` that does the run with $PLATFORM-$ARCH etc so that you could write instead:

```
machine config -h
```

So you won't have to repeat this every time.